modules/overlays/test/overlays/html-tooltip-overlay.test.js

```diff
@@ -0,0 +1,17 @@
+// @flow-ignore
+import React from 'react';
+import Renderer from 'react-test-renderer';
+
+import { viewport } from '../../../../core/src/test/mocks';
+import Nebula from '../../lib/nebula-react';
+import HtmlTooltipOverlay from '../../lib/overlays/html-tooltip-overlay';
+
+it('test HtmlTooltipOverlay no items shown', () => {
+  const Component = (
+    <Nebula viewport={viewport} layers={[]}>
```

Comment:
Q: Should we just use `DeckGL` here?

Reply:
Yes, let me fix that and add more unit tests.